index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <title>Portfolio</title>
+</head>
+<body class="body">
+
+<header class="toolbar">
+    <div class=left>
+        <p class=my-name>Mouaz El Molkey</p>
+         </div>
+        
+         <div class=right>
+            <a class=menu_item>Portoflio</a>
+            <a class=menu_item>About</a>
+            <a class=menu_item>Contact</a>
+            <img class="menu_mail" src="/img/mail.png" alt="Email">
+             </div>
+            
+</header>
+<section class="intro">
+<h1 class="big-name"> Mouaz El Molkey</h1>
+<img src="img/circle-1.png" class="circle-1" alt="">   
+<h2 class="small-name">Software Developer</h2>
+</section>
+
+<section class="main-section">
+    <h3 class="can-do">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</h3>
+    <img src="img/circle-2.png" class="circle-2" alt=""> 
+<ul class="main">
+<li class="main-list">
+    
+ <a class="skill-name">Language</a>
+
+<div class="box">
+<a class="items" href="#">Javascript</a>
+<a class="items" href="#">Ruby</a>
+<a class="items" href="#">Html</a>
+<a class="items" href="#">Css</a>
+</div>
+</li>
+
+<li class="main-list">
+    <a class="skill-name">Frameworks</a>
+
+<div class="box h-box">
+<a class="items" href="#">Bootstrap</a>
+<a class="items" href="#">Ruby on Rail</a>
+<a class="items" href="#">RSPec</a>
+<a class="items" href="#">CapyBara</a>
+<a class="items" href="#">Selenium</a>
+</div>
+</li>
+
+<li class="main-list">
+<a class="skill-name">Skills</a>
+<div class="box">
+<a class="items" href="#">Codekit</a>
+<a class="items" href="#">Github</a>
+<a class="items" href="#">Codepen</a>
+<a class="items" href="#">Gitlab</a>
+<a class="items" href="#">Terminal</a>
+</div>
+</li>
+</ul>
+</section>
+
+<section class="social">
+<img class="social-items" src="/img/github.png" alt="">
+<img class="social-items" src="/img/linkedin.png" alt="">
+<img class="social-items" src="/img/vic.png" alt="">
+<img class="social-items" src="/img/twitter.png" alt="">
+<img class="social-items" src="/img/micro.png" alt="">
```

Comment:
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Please fix that.